packages/itwin/one-click-lca-widget/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@itwin/one-click-lca-widget",
```

Comment:
I think we need a different package name, this doesn't fit with out current naming schema, itwin/xyz-react

Reply:
@calebmshafer , i dont love the new name either, but thoughts?

Reply:
how do we feel about one-click-lca-react? still quite a mouthful, so oclca-react?

Reply:
oclca-react sounds like a good option to me
I doubt people will be looking for the widget without already knowing what one click lca is